### ISSUE TITLE
Amount not changed after validation

### ISSUE THREAD

**Issue by Stef-Feanx** [moneymanagerex/moneymanagerex#2424]
- Check OS
  - [X] Windows
  - [ ] Mac OSX
  - [ ] Linux 
-  Check MMEX version
  - [X] 1.3.4
  - [ ] 1.2.x
  - [ ] 1.1 or older

Just installed the last version :
1/ the installation folder in Program Files is not the same than in version 1.3.3 !!! WHY ?
2/ I have corrected an amount in a line : I have changed 49.99 into 54.99. Clicked on OK.
The amount is NOT changed on the main display AND the total si still the same !
Thank you !

**Comment by vomikan** [comment]
1) Befor it was mmex folder? Minor issue.
![image](https://user-images.githubusercontent.com/6836805/80311300-8bbf8100-87e7-11ea-81fe-3a9c4b3b4f28.png)
2) I can't reproduce this. Please follow instructions in the app Help - Report a Bug

**Comment by Stef-Feanx** [comment]
Thank for the answer.
1/ I haven't checked on install, I have supposed that the folder was the same. Thank you !
2/ I have double-clicked on an amount to correct it. In the opened box, I have corrected it and clicked on OK. On the main display, the amount has not changed. If I re-open the box, the amount is well corrected ! I have tried several times on several lines : same thing, the main page does'nt display any change... I will try to re-install it on another computer and cjek again...

**Comment by vomikan** [comment]
Could you please provide a screenshot? May be we talking about different boxes.

**Comment by Stef-Feanx** [comment]
I have uninstalled both versions 1.3.3 and 1.3.4.
I have reinstalled 1.3.4. reproducing the error of folder (voluntary)
I seems to work fine this time :-/
I think this problem was from the protection of files from BitDefender which has blocked the new MoneyManager on the first use... I tried on another computer with the same configuration and it also worked well...
I think this bug can be closed. sorry !
